### ISSUE TITLE
Falcon Server: Cannot find module 'lodash/get'

### ISSUE THREAD

**Issue by prosellen** [deity-io/falcon#310]
**Describe the bug**
After creating a new app, the server exits with the uncaught exception "Cannot find module 'lodash/get'"

**To Reproduce**
1. Create new app with `yarn create falcon-app falcon`
2. `cd C:\Users\<user>\projects\falcon\server`
3. `yarn start`
4. Error:
```yarn run v1.13.0
$ cross-env NODE_ENV=development nodemon index.js
[nodemon] 1.18.10
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node index.js`
Uncaught Exception:  { Error: Cannot find module 'lodash/get'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\peter.rosellen\projects\deity-falcon\server\node_modules\@deity\falcon-server\src\index.js:5:13)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3) code: 'MODULE_NOT_FOUND' }
[nodemon] clean exit - waiting for changes before restart
```

**Expected behavior**
Server should start

**Desktop:**
 - OS: Windows 10
 - Node: v10.15.1
 - Yarn: 1.13.0
 - create-falcon-app: 1.1.5

**Comment by cane-lagerwaard** [comment]
Hi @prosellen thanks for your feedback. We're currently trying to reproduce your issue. In the meantime, could you try creating the app with `npx create-falcon-app my-app` instead of `yarn` and let us know if you're encountering the same issue?

**Comment by prosellen** [comment]
Hi @cane-lagerwaard, thanks for the fast reply.

I can confirm the same issue after ```npx create-falcon-app my-app```.

With `npm start`:
```
> @deity/example-shop-with-blog-server@0.2.1 start C:\Users\peter.rosellen\projects\my-app\server
> cross-env NODE_ENV=development nodemon index.js

[nodemon] 1.18.10
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node index.js`
Uncaught Exception:  { Error: Cannot find module 'lodash/get'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\peter.rosellen\projects\my-app\server\node_modules\@deity\falcon-server\src\index.js
:5:13)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3) code: 'MODULE_NOT_FOUND' }
[nodemon] clean exit - waiting for changes before restart
```

With `yarn start`
```
yarn run v1.13.0
$ cross-env NODE_ENV=development nodemon index.js
[nodemon] 1.18.10
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node index.js`
Uncaught Exception:  { Error: Cannot find module 'lodash/get'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\peter.rosellen\projects\my-app\server\node_modules\@deity\falcon-server\src\index.js:5:13)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3) code: 'MODULE_NOT_FOUND' }
[nodemon] clean exit - waiting for changes before restart
```

**Comment by prosellen** [comment]
As and added note: the client also encouters an exception when referencing lodash: #311

**Comment by cane-lagerwaard** [comment]
@prosellen could you try manually deleting the `node_modules` folders within `/client` and `/server` and reinstalling them with `npm install` or `yarn install`

**Comment by Borales** [comment]
@prosellen could you please share with us the output of `npx create-falcon-app my-app` command?

**Comment by prosellen** [comment]
Sure thing. Here you go, @Borales :
```
 √ Generating base structure for my-app App
 √ Installing dependencies

Application created successfully!
All required dependencies were installed, so you don't need to worry about them.

Generated my-app application contains 2 sub-applications
Please run the following code in separate terminals to start your applications in development mode

cd C:\Users\peter.rosellen\projects\my-app\client
yarn start

cd C:\Users\peter.rosellen\projects\my-app\server
yarn start

```

**Comment by prosellen** [comment]
@cane-lagerwaard 

I removed the `node_modules' folder and ran `npm i`. Here's the output:
```
> protobufjs@6.8.8 postinstall C:\Users\peter.rosellen\projects\my-app\server\node_modules\protobufjs
> node scripts/postinstall

> nodemon@1.18.10 postinstall C:\Users\peter.rosellen\projects\my-app\server\node_modules\nodemon
> node bin/postinstall || exit 0

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN @apollographql/apollo-upload-server@5.0.3 requires a peer of graphql@^0.13.1 but none is installed. You must install peer dependencies yourself.
npm WARN apollo-tracing@0.3.3 requires a peer of graphql@0.10.x - 14.0.x but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 454 packages from 354 contributors and audited 3967 packages in 52.407s
found 0 vulnerabilities
```

After that, `npm start` produces a new error: now the module  `'core-js/proposals/array-flat-and-flat-map'` cannot be found:
```
> @deity/example-shop-with-blog-server@0.2.1 start C:\Users\peter.rosellen\projects\my-app\server
> cross-env NODE_ENV=development nodemon index.js

[nodemon] 1.18.10
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node index.js`
Uncaught Exception:  { Error: Cannot find module 'core-js/proposals/array-flat-and-flat-map'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (C:\Users\peter.rosellen\projects\my-app\server\node_modules\apollo-env\lib\polyfills\array.js
:3:1)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3) code: 'MODULE_NOT_FOUND' }
[nodemon] clean exit - waiting for changes before restart
```

Manually installing graphql@^0.13.1 (as suggested by the output from `npm i`) does not change this.

**Comment by cane-lagerwaard** [comment]
After doing some digging around I've found a temporary workaround. You can do `npm install core-js@3.0.0-beta.11` within `client` and `server` and it should work. We'll be releasing a hotfix soon

**Comment by prosellen** [comment]
Thank you. Installing `core-js@3.0.0-beta.11` in `server` works for me.  

Actually, it also fixes the `lodash/get` error in the process. So, until the hotfixes arrives, the fastest way to get the setup working for me was:
1. `npx create-falcon-app my-app`
1. `cd my-app\server`
1. `npm install core-js@3.0.0-beta.11`
1. `npm start`

**Comment by Borales** [comment]
Hi @prosellen 
The fix has been merged in #313

**Comment by cane-lagerwaard** [comment]
Closing this issue as it is fixed in Falcon v1.0-rc
